copier.yml

```diff
@@ -53,11 +53,8 @@ license:
 
 dbt_target:
   choices:
```

Comment:
Should I remove this "choices" key from here? @ian-r-rose

Reply:
Yeah, we can remove the `dbt_target` section all togehter

Reply:
And all references to `dbt_target` in this project can be removed

Reply:
Got it, thank you!